msteams-platform/whats-new.md

```diff
@@ -1,9 +1,8 @@
 ---
 title: What's new
-description: Describes all the new or latest developer features in Microsoft Teams
+description: Learn about all the new or latest developer features and improvements in Microsoft Teams
```

Comment:
We aren't learning about improvements in Teams in this article. I'm editing this accordingly.
CC @surbhigupta12 @v-npaladugu